modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,143 @@ This release seeks to improve developer agility, reduce total cost of ownership,
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-652]
+== Release 6.5.2 (February 2021)
+
+Couchbase Server 6.5.2, released in February 2021, is the second maintenance release in the 6.5.x series for Couchbase Server. This release includes important bug fixes in various components.
+
+[#fixed-issues-652]
+=== Fixed Issues
+
+==== Analytics Service
+
+[#table_fixedissues_v652-analytics,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-44220[MB-44220^]
+| *Summary*: Fixed an issue where the Analytics driver went into a restart loop when a filesystem corruption was encountered, resulting in the Analytics service exiting with an error.
+
+| https://issues.couchbase.com/browse/MB-44215[MB-44215^]
+| *Summary*: The validity bit was used to determine corrupted merged components. 
+
+| https://issues.couchbase.com/browse/MB-44214[MB-44214^]
+| *Summary*: When making the debug log async, the DCP client log level was incorrectly set to ERROR. Fixed this issue to use the global service log level instead, which defaults to DEBUG.
+|===
+
+==== Cluster Manager
+
+[#table_fixedissues_v652-cluster-manager,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-42607[MB-42607^]
+| *Summary*: During TLS handshake with the Data Service, if the node certificate required more than 8192 bytes to transmit, then the handshake failed. This has been fixed.
+|===
+
+==== Data Service
+
+[#table_fixedissues_v652-data,cols="25,66"]
+|===
+| Issue | Description
```

Comment:
As an aside, is there any particular order to the MBs listed here? If not, I'd suggest sorting by number to aid the customer in finding things (and also to make it easier to review them!)

Reply:
Typically, we group MBs by component and within each group, they are sorted in descending order.

Reply:
Ah yes, sorry I thought they were not in numerical order but I think I was looking across different tables :)